plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -845,11 +845,49 @@ func PodFitsHostPorts(pod *v1.Pod, meta algorithm.PredicateMetadata, nodeInfo *s
 	}
 
 	existingPorts := nodeInfo.UsedPorts()
```

Comment:
I can not remember why we can not just delete the port from map, can you help to check it?

Reply:
I don't quite sure about it either, you wrote it in https://github.com/kubernetes/kubernetes/pull/42524

Reply:
@k82cn I can open another pr to change the logic, I think delete the port from map is better

Reply:
that's ok to me.

Reply:
ok

Reply:
I think this problem is fixed by https://github.com/kubernetes/kubernetes/pull/54538, so no longer a problem